src/monster.h

```diff
@@ -4,6 +4,7 @@
 #include "creature.h"
 #include "enums.h"
 #include "int_id.h"
+#include "field.h"
```

Comment:
I think I'll go the forward declarations route; I'm pretty adverse to depending on invisible includes of includes if they are at all avoidable (even if they are required now by something else, it's still hypothetically possible that may change later and the changer may miss the include dependency).